bookkeeper-server/src/main/java/org/apache/bookkeeper/client/RackawareEnsemblePlacementPolicyImpl.java

```diff
@@ -240,7 +237,16 @@ public void reloadCachedMappings() {
         help = "The distribution of number of bookies reordered on each read request"
     )
     protected OpStatsLogger readReorderedCounter = null;
+    @StatsDoc(
+            name = FAILED_TO_RESOLVE_NETWORK_LOCATION_COUNTER,
+            help = "Counter for number of times DNSResolverDecorator failed to resolve Network Location"
+    )
     protected Counter failedToResolveNetworkLocationCounter = null;
+    @StatsDoc(
+            name = NUM_BOOKIES_IN_DEFAULT_FAULTZONE,
+            help = "Gauge for the number of Bookies in default Fault Zone"
+    )
+    protected Gauge<Integer> numBookiesInDefaultFaultZone;
```

Comment:
we rarely use "fault zone" in bookkeeper's concept. I would prefer using a consistent name in the same file, e.g. calling it default rack.

Reply:
ok will rename it to defaultrack